recirq/qcqmc/trial_wf.py

```diff
@@ -212,6 +213,97 @@ def _json_dict_(self):
         simple_dict["params"] = self.params
         return simple_dict
 
+    @classmethod
+    def build_pp_plus_trial_from_dependencies(
+        cls,
+        params: PerfectPairingPlusTrialWavefunctionParams,
+        *,
+        dependencies: Dict[data.Params, data.Data],
+        do_print: bool = False,
+    ) -> "TrialWavefunctionData":
+        """Builds a TrialWavefunctionData from a TrialWavefunctionParams
+
+        Args:
+            params: The parameters specifying the PP+ trial wavefunction.
+            dependencies: Data dependencies
+            do_print: Print debugging information to stdout
+
+        Returns:
+            The constructed TrialWavefunctionData object.
+        """
+
+        # Avoid circular imports
+        from recirq.qcqmc import optimize_wf
```

Comment:
This is a code smell that indicates that we may want to further split up these files in order to remove this.  You can do this in a separate PR if you want.

Reply:
Agreed. I'll punt to a follow up.